src/uct/ib/base/ib_verbs.h

```diff
@@ -113,6 +113,17 @@
 #  define IBV_DEVICE_HAS_NOP(_attr)                 0
 #endif /* HAVE_DECL_IBV_EXP_WR_NOP */
 
+/*
+ * Adaptive Routing support
+ */
+#if HAVE_DECL_IBV_EXP_QP_OOO_RW_DATA_PLACEMENT
+#  define UCX_IB_DEV_IS_OOO_SUPPORTED(_ibdev, _transport)  \
```

Comment:
let's make this macro accept _attr and not _ibdev like rest of the macros in this file